src/test/java/core/basesyntax/sarvice/impl/FruitTransactionParserServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.sarvice.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitTransactionParserServiceImpl;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserServiceImplTest {
+    private static FruitTransactionParserServiceImpl parser;
+    private List<FruitTransaction> expected;
+
+    {
+        expected = new LinkedList<>();
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.BALANCE, "banana", 20));
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.BALANCE, "apple", 100));
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.SUPPLY, "banana", 100));
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/191004652-50829ea9-39e0-4bf8-81c0-4e2d71ec455e.png)
here you initialize in the initialization block, in StorageDaoTest in method - choose 1 option)